sdk/storage/storage-blob/test/perfstress/track-1/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "track-1",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@azure/storage-blob": "^10.5.0",
+    "@azure/test-utils-perfstress": "file:../../../../../test-utils/perfstress/azure-test-utils-perfstress-1.0.0.tgz",
```

Comment:
How does this work?  Is the tgz checked into the repo?

Reply:
No, it's not checked into the repo.

This "track-1" perf test folder is not managed by rush(as rush doesn't recommend nested projects).

Installing perf framework into this track-1 folder has to be done manually by referring to the instructions in the readme.
https://github.com/Azure/azure-sdk-for-js/blob/5885e90120502cf0eb25920b6469ffcb3ccbf7f8/sdk/storage/storage-blob/test/perfstress/track-1/README.md

For getting the perf package in this folder, one needs to build the perf project and install it here.

Since this is track-1, it should be fine as we only intend to run them once for the baseline metrics. (And since we'd be getting rid of the track-1 folders soon.)

We can have a script that does the work needed based on the instructions in the readme. Would you suggest such or do you have any other thoughts?

Reply:
This sounds like a reasonable compromise since Track1 doesn't use rush.